lib/intl.mjs

```diff
@@ -41,21 +43,44 @@ const descriptor = (value) => {
 const IntlDateTimeFormat = globalThis.Intl.DateTimeFormat;
 const ObjectAssign = Object.assign;
 
-export function DateTimeFormat(locale = IntlDateTimeFormat().resolvedOptions().locale, options = {}) {
+// Construction of  built-in Intl.DateTimeFormat objects is sloooooow,
+// so we'll only create those instances when we need them.
+// See https://bugs.chromium.org/p/v8/issues/detail?id=6528
+function getPropLazy(obj, prop) {
+  let val = obj[prop];
+  if (prop === TZ_RESOLVED) {
```

Comment:
I'd probably split this part into a separate `getResolvedTimeZoneLazy()` function, so that we have two lazy property retrieval functions that have clear input and return types. That will probably help when converting the code to TypeScript.

Reply:
Good idea. Done.